adapters/triplelift_native/triplelift_native.go

```diff
@@ -89,7 +99,7 @@ func (a *TripleliftNativeAdapter) MakeRequests(request *openrtb2.BidRequest, ext
 	var validImps []openrtb2.Imp
 	// pre-process the imps
 	for _, imp := range tlRequest.Imp {
-		if err := processImp(&imp); err == nil {
+		if err := processImp(&imp, request); err == nil {
```

Comment:
I think you need to pass in a pointer to the request (&request) and not the request itself

Reply:
I believe this is ok. We already passing a pointer (via the MakeRequests arg)